CHANGELOG.md

```diff
@@ -35,6 +39,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 https://tls.mbed.org/tech-updates/releases/mbedtls-2.16.5-and-2.7.14-released)
 release notes for the set of issues addressed.
 
+### Deprecated
+- In the next release (v0.10), system EDL will no longer be compiled into OE
+  libraries by default (COMPILE_SYSTEM_EDL will be OFF by default). See the
+  [system EDL opt-in document]
+  (docs/DesignDocs/system_ocall_opt_in.md#how-to-port-your-application) for
+  more details on how to rebuild the SDK to match this behavior and for
+  guidance on porting your application to the new model.
```

Comment:
Do we have a GitHub issue filed to track porting all samples over to the new model?

Reply:
#2867